setup.py

```diff
@@ -2,7 +2,7 @@
 from setuptools import setup
 
 setup(name='python-figo',
-      version='1.6.0',
+      version='1.6.1',
```

Comment:
The `USER_AGENT` still contains `1.5.4`. I guess we want one constant somewhere that is used by this `setup()` call and to build the `USER_AGENT`, so it has to be changed only once on new releases.